core/conversion/converters/impl/interpolate.cpp

```diff
@@ -11,12 +14,75 @@ namespace converters {
 namespace impl {
 namespace {
 
+/*
+ * Helper functions
+ */
+
+auto parse_nearest(args args) {
+    auto in = args[0].ITensor();
+    auto in_shape = util::toVec(in->getDimensions());
+
+    return  std::make_tuple(in, in_shape);
+}
+
+auto parse_linear(args args) {
+    auto in = args[0].ITensor();
+    auto in_shape = util::toVec(in->getDimensions());
+    bool align_corners = args[2].unwrapToBool();
+
+    return  std::make_tuple(in, in_shape, align_corners);
+}
+
+void create_plugin(ConversionCtx* ctx, const torch::jit::Node* n, nvinfer1::ITensor* in, const char* name, 
+                                                                                         std::vector<int64_t> in_shape, 
+                                                                                         std::vector<int64_t> out_shape, 
+                                                                                         std::vector<int64_t> out_size, 
+                                                                                         std::string mode) {
+    auto creator = new plugins::InterpolatePluginCreator();
+    auto plugin = creator->createPlugin(name, in_shape, out_shape, out_size, mode, false);
+
+    auto resize_layer = ctx->net->addPluginV2(reinterpret_cast<nvinfer1::ITensor* const*>(&in), 1, *plugin);
+    TRTORCH_CHECK(resize_layer, "Unable to create interpolation plugin from node" << *n);
+
+    resize_layer->setName(util::node_info(n).c_str());
+
+    auto layer_output = ctx->AssociateValueAndTensor(n->outputs()[0], resize_layer->getOutput(0));
+
+    LOG_DEBUG("Output tensor shape: " << layer_output->getDimensions());

```

Comment:
Can you add a warning here that this layer will be run with ATen and not TensorRT so people know why performance may be different?